examples/quick_start_with_docker/run_detailed.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+echo "Enter AWS ACCESS KEY"
+read aws_access_key
```

Comment:
100% agree, this was more of a first pass to see what value it can provide to the project.